### PR TITLE
adds classnotes 2018-9-24 and updates link on main page

### PR DESCRIPTION
**Pull request by devinepk**

### CODE REVIEW

File: notes/2018-09-24.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>September 24, 2019</title>
```
Comment (bmizepatterson) [inline notes/2018-09-24.html]: Don't think it's 2019 quite yet ;)

**Comment by Kiwipup** [comment]
looks good! +1

File: notes/2018-09-24.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    <link href="../style.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+    <div id="nav">
+        <a href="../index.html">Home</a>
+    </div>
+
+    <h1>September 24, 2019</h1>
```
Comment (bmizepatterson) [inline notes/2018-09-24.html]: 2018

**Comment by bmizepatterson** [comment]
Just fix the date to 2018 and it looks good :)

File: notes/2018-09-24.html (hunk 1 above)
Comment (devinepk) [reply notes/2018-09-24.html]: thanks 👍

**Pull request merged**